AditiBeautyCare/AditiBeautyCare.Business/Common/Sample/RequestResult.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using AditiBeautyCare.Business.Common.Sample;
+namespace AditiBeautyCare.Business.Common.Sample
+{
+    public class RequestResult<T> : ResultBase
```

Comment:
please put comments here to explain the purpose of the method

Reply:
added the comment as asked